.vib/cassandra/goss/goss.yaml

```
@@ -0,0 +1,31 @@
+file:
+  {{ .Vars.persistence.mountPath }}:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/cassandra/conf/cassandra.yaml:
+    exists: true
+    filetype: file
+    mode: "0644"
+    group: root
+    contains:
+      - /num_tokens.*{{ .Vars.cluster.numTokens }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.create }}
```

I think the right expression would be:

```suggestion
    exists: {{ and .Vars.serviceAccount.create .Vars.serviceAccount.automountServiceAccountToken }}
```

The serviceAccount can be created regardless of the token being mounted.

---

Good catch, fixed!